app/src/main/java/com/duckduckgo/app/global/view/FireDialog.kt

```diff
@@ -140,7 +142,12 @@ class FireDialog(
         }
     }
 
-    private fun animationEnabled() = settingsDataStore.fireAnimationEnabled
+    private fun animationEnabled() = settingsDataStore.fireAnimationEnabled && animatorDurationEnabled()
+
+    private fun animatorDurationEnabled(): Boolean {
+        val animatorScale = Settings.Global.getFloat(context.contentResolver, ANIMATOR_DURATION_SCALE, 1.0f)
```

Comment:
I think default value should not be `1f` but rather `0.0f` To fall on the safe side when the value is not defined.

Reply:
Good spot @aitorvs, I've made the appropriate change.